pytorch_lightning/utilities/cli.py

```diff
@@ -109,9 +107,9 @@ def __init__(self, *args: Any, parse_as_dict: bool = True, **kwargs: Any) -> Non
         if not _JSONARGPARSE_AVAILABLE:
             raise ModuleNotFoundError(
                 "`jsonargparse` is not installed but it is required for the CLI."
-                " Install it with `pip install jsonargparse[signatures]`."
+                " Install it with `pip install -U jsonargparse[signatures]`."
```

Comment:
Can you also write

```python
_JSONARGPARSE_AVAILABLE = _module_available("jsonargparse") and _compare_version("jsonargparse", operator.ge, "4.0.0")
```

in https://github.com/PyTorchLightning/pytorch-lightning/blob/5788789f0164b984fdcb9d6390b3babbf5997fec/pytorch_lightning/utilities/imports.py#L88

I was going to commit this myself but I'm getting

```console
$ gh pr checkout 10426
From https://github.com/PyTorchLightning/pytorch-lightning
 ! [rejected]            refs/pull/10426/head -> jsonargparse-4-backward-compatible  (non-fast-forward)
exit status 1
```

Reply:
Done. Sorry about the reject. Originally this pull request was only for checking backward compatibility with 1.5. The deprecation changes I had in another branch so overwrote this branch with the other.